docs/participate/online-communities.mdx

```diff
@@ -41,10 +41,10 @@ The [Mina Community Blog](https://minafoundation.notion.site/minafoundation/Mina
 
 The Mina community and its members are the core of Mina's global, decentralized network. You can be an active contributor to collective brilliance. For inspiration and steps on how to contribute to the experimental community blog, see [Open Call for Mina Community Creators](https://minaprotocol.com/blog/mina-community-blog).
 
-## Ecosystem Partners
+## Mina Ecosystem Partners
 
-- [O(1) Labs Blog](https://blog.o1labs.org/) - Incubators of the Mina Protocol 
-- [=nil; Foundation Blog](https://blog.nil.foundation/tag/mina/) - Creators of the zkBridge bridge from Mina to Ethereum
-- [Viable Systems' OpenMina Blog](https://medium.com/openmina) - Tools to help improve the performance of the Mina nodes and overall network
+- O(1) Labs [Blog](https://blog.o1labs.org/) - Incubators of the Mina Protocol 
+- =nil; Foundation [Blog](https://blog.nil.foundation/tag/mina/) - Creators of the zkBridge bridge from Mina to Ethereum
+- Viable Systems [OpenMina Blog](https://medium.com/openmina) and [CEO Juraj Selep Blog](https://medium.com/@jurajselep) - Tools to help improve the performance of the Mina nodes and overall network
```

Comment:
@jasongitmail wdyt about adding link to https://medium.com/@jurajselep as suggested on [Slack](https://o1-labs.slack.com/archives/C029LQN4K9U/p1679519886244419?thread_ts=1679518984.661459&cid=C029LQN4K9U) many of his Mina posts are not on the OpenMina blog

Reply:
Given he has only 2 Mina posts, and only one in 'Open Mina' (so it's not super useful to link to that), and the rest are Tezos related, would it be worth linking to this query instead: https://medium.com/search?q=juraj+selep+mina+protocol?  

It might be unusual to use a query, but it returns exactly the desired content and no more.

thoughts?

Reply:
hmmm we might be setting a weird precedent in this case, let's just omit his personal blogs and include official ecosystem partner blog links until we receive further input?

Reply:
True, but you could consider that the functional equivalent of linking to a 'mina' tag. 

But sounds good to me to exclude it too, given there are only 2 Mina posts total and only 1 tagged.